build.sbt

```diff
@@ -212,7 +214,15 @@ lazy val server: Project = (project in file("server"))
       credentials += Credentials(Path.userHome / ".s3_elasticmq_credentials"),
       // docker
       dockerExposedPorts := Seq(9324,9325),
-      dockerBaseImage := "openjdk:8u212-b04-jdk-stretch",
+      dockerBaseImage := "openjdk:11-jdk-stretch",
```

Comment:
shouldn't we do the same for `nativeServer`?

Reply:
According to comments in the issue - `nativeServer` works on ARM-based machines.